src/wheel/util.py

```diff
@@ -8,6 +8,9 @@
 
     StringIO = io.BytesIO
 
+    def TextIOWrapper(bytes_io):
```

Comment:
Why the separate code path on py2 (not that we care about py2 much anymore, but the codebase still supports it)?

Reply:
Well `ZipFile.open` returns a `BytesIO` but `csv.reader` expects a `TextIO`. To smooth this over, I use the `TextIOWrapper` which converts a `BytesIO` into a `TextIO`. In Python 2, there is no difference between `str` and `bytes` so there is no difference between `TextIO` and `BytesIO`. So I assumed incorrectly that `io.TextIOWrapper` would not be a thing.